### PR TITLE
Feature/us1668 warning on page nav

### PR DESCRIPTION
**Pull request by IngageMaddoxT**

### CODE REVIEW

File: crossroads.net/app/my_serve/myserve.controller.js [hunk 1]
```diff
@@ -63,6 +83,20 @@
       vm.filterState.clearAll();
     });
 
+    $window.onbeforeunload = function(){
```
Comment (IngageMaddoxT) [inline crossroads.net/app/my_serve/myserve.controller.js]: added these two functions

File: crossroads.net/app/my_serve/myserve.html [hunk 2]
```diff
@@ -18,11 +19,14 @@ <h1 class="page-header">Sign Up To Serve </h1>
       <span ng-bind-html="$root.messages[$root.MESSAGES.noServingOpportunities].message | html"></span>
     </div>
 
+    <form name='serveForm'>
```
Comment (IngageMaddoxT) [inline crossroads.net/app/my_serve/myserve.html]: added a form so we have to check for dirty

File: crossroads.net/app/my_serve/myserve.controller.js [hunk 3]
```diff
@@ -15,20 +35,20 @@
...
-    vm.loadText = "Load More";
+    vm.loadText = 'Load More';
     vm.original = [];
     vm.showButton = showButton;
     vm.showNoOpportunitiesMsg = showNoOpportunitiesMsg;
 
     activate();
 
-    $rootScope.$on("personUpdated", personUpdateHandler);
+    $rootScope.$on('personUpdated', personUpdateHandler);
```
Comment (charlie-retzler) [inline crossroads.net/app/my_serve/myserve.controller.js]: Should all of the $rootsScope.$on be named functions?

**Comment by charlie-retzler** [comment]
:+1:

**Pull request merged**